### PR TITLE
added buildCoverageRegex and buildGitStrategy to Project

### PR DESCRIPTION
**Pull request by kvakos**
adds two attributes to Project

### CODE REVIEW

File: src/main/java/org/gitlab4j/api/models/BuildGitStrategy.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    @JsonValue
+    public String toValue() {
+        return (enumHelper.toString(this));
+    }
+
+    @Override
+    public String toString() {
+        return (enumHelper.toString(this));
+    }
+}
```
Comment (gmessner) [inline src/main/java/org/gitlab4j/api/models/BuildGitStrategy.java:53]: @kvakos Unless an enum is shared by multiple models we usually will not create a stand-alone class for it, please move this into ```org.gitlab4j.api.Constants``` (preferred) or directly into ```org.gitlab4j.api.models.Project```

Comment (kvakos) [reply src/main/java/org/gitlab4j/api/models/BuildGitStrategy.java]: BuildGitStrategy is now inner class  of Project

Comment (kvakos) [reply src/main/java/org/gitlab4j/api/models/BuildGitStrategy.java]: I changed to preferred implementation.

**Comment by gmessner** [comment]
@kvakos Release 4.15.4 is now available with this functionality.

**Pull request merged**